jenkins/scripts/cbts/coverage_utils/stability_diff.py

```
@@ -0,0 +1,156 @@
+        if not byte:
+            continue
+        base = byte_idx * 8
+        for bit in range(8):
+            if byte & (1 << bit):
+                out.append(base + bit)
+    return out
+
+
+def main(argv):
+    if len(argv) != 4:
+        print(__doc__, file=sys.stderr); return 2
```

Ruff (0.15.15)

[error] Multiple statements on one line (semicolon)

(E702)